resize.c

```diff
@@ -42,50 +43,95 @@
 #endif
 #endif
 
-/**
- * mutt_resize_screen - Called after receiving SIGWINCH
- */
-void mutt_resize_screen(void)
+int mutt_resize_getrows(int rows)
 {
   const char *cp = NULL;
-  int fd;
-  struct winsize w;
-#if !defined(USE_SLANG_CURSES)
-  int SLtt_Screen_Rows, SLtt_Screen_Cols;
-#endif
+  cp = mutt_str_getenv("LINES");
+  if (cp && (mutt_str_atoi(cp, &rows) < 0))
+    return 24;
+  return -1;
```

Comment:
The default of `-1` **is** the original behaviour,
but I think returning a sensible value (24) is a better idea.